tools/debian/cockpit-ws.postinst

```diff
@@ -3,6 +3,8 @@ set -e
 
 adduser --system --group --home / --no-create-home cockpit-ws
 
+rm -f /etc/issue.d/cockpit
```

Comment:
This needs to be done more cautiously. In particular, it should be version-guarded, should verify that the file is a symlink, and add a comment:
```
# file got renamed to cockpit.issue
if dpkg --compare-versions "$2" lt-nl "176-1~" && [ -L /etc/issue.d/cockpit ]; then
     rm -f /etc/issue.d/cockpit
fi
```
Usually one uses `dpkg-maintscript-helper(1)` for this, i. e. add a debian/cockpit-ws.maintscript with
```
rm_conffile /etc/issue.d/cockpit 176-1
```
If that works, it's preferrable over writing the postinst code manually, as it also takes care of aborted upgrades and is less error prone. But I'm not sure if that gets along with a symlink, so please test this.

Reply:
I'll take a look, but note: the file /etc/issue.d/cockpit is *never* useful.  This file will always be ignored.  That's why I'm being not-so-careful here.

Reply:
So, I've discovered that debian actually removes symlinks from /etc on uninstall with no further help, so there, at least, we don't need to do anything.